### PR TITLE
DOS-2738: [CONTROLLER] Fix fleet line length limit bug

### PR DESCRIPTION
**Pull request by aj-may**
Still need to test this.

### CODE REVIEW

File: controller/scheduler/fleet.py [hunk 1]
```diff
@@ -187,6 +180,12 @@ def _create_container(self, name, image, command, unit, **kwargs):
         # encode command as utf-8
         if isinstance(l.get('command'), basestring):
             l['command'] = l['command'].encode('utf-8')
+
+        values = kwargs.get('values', {})
+        if values:
+            for key, value in values.items():
+                unit.append({"section": "Service", "name": "ExecStartPre", "value": '''/bin/sh -c "echo '{key}={value}' >> /tmp/env_files/{name}"'''.format(key=key, value=value)})
```
Comment (DanHoerst) [inline controller/scheduler/fleet.py]: Is it possible to put these files in `/etc/env_files/{name}` - which should be writable by the user creating these unit files

**Comment by DanHoerst** [comment]
Fyi assuming this works we will need the commit messages as descriptive as possible (see https://github.com/Hearst-Hatchery/deis/pull/15) in order to have the fork re-certified.

**Comment by gabrtv** [comment]
Please make sure you run it through proper testing, as we have no ability to test this properly.  That said, the Deis team is happy to certify this change as-is. 👍

**Comment by aj-may** [comment]
@gabrtv 

We have tested the following scenarios:
- deploying an app with 100+ vars
- scaling the app up and down
- adding and removing env vars
- deploying an app with no env vars
- destroying and recreating an application

If you want to, you should be able to run test scenarios yourself by following these steps:
1. Deploy a deis cluster with our custom controller (quay.io/hearst/deis-controller:h1.12.2-37-f3a712d)
2. Create an application with many env vars.
3. profit

**Comment by gabrtv** [comment]
Step 3 sounds nice.. That testing looks good to me.  You have our go-ahead to merge!

**Pull request merged**